source/_components/homematicip_cloud.markdown

```diff
@@ -65,6 +65,8 @@ Add the information to your `configuration.yaml` file:
 homematicip_cloud:
   - accesspoint: IDENTIFIER
     authtoken: AUTHTOKEN
+    enable_group_switches: true
```

Comment:
This is an optional variable and should not be included in the first example yaml config.